x-pack/plugins/apm/public/components/app/onboarding/commands/django.ts

```diff
@@ -48,9 +43,6 @@ ELASTIC_APM = {
 
   # {{{serverUrlHint}}}
   'SERVER_URL': '{{{apmServerUrl}}}',
-
-  # {{{serviceEnvironmentHint}}}
-  'ENVIRONMENT': 'my-environment',
```

Comment:
Same with `environment`: we still want to show that in the code snippet. But there's no reason to show the boilerplate value in the table.

You could change it like this to make it a little more clear:

```
'ENVIRONMENT': '<your-environment>',
```

Reply:
@sqren done 👍🏼